Move well known spec values into appendix (#411)
foxglove/mcap: docs/specification/appendix.md

# MCAP File Format Specification Appendix

This document describes well known values for MCAP files.

## Well-known compression formats

The Chunk `compression` field may contain the following options:

- [lz4](<https://en.wikipedia.org/wiki/lz4_(compression_algorithm)>): an algorithm that prioritizes compression/decompression speed over compression ratio.

> - [zstd](https://en.wikipedia.org/wiki/zstandard): an algorithm that prioritizes compression ratio over compression/decompression speed.

Reviewer: I realize this text has probably read this way for a while, but I think a better description for zstd is “an algorithm with a tunable compression ratio/speed tradeoff”.

> - [zstd](https://en.wikipedia.org/wiki/zstandard): an algorithm that prioritizes compression ratio over compression/decompression speed.

Reviewer 2: ```suggestion
- [zstd](https://en.wikipedia.org/wiki/zstandard): an algorithm with a tunable compression ratio/speed tradeoff.
```
